Turtlelord26 #491 explosive damage consistency

Turtlelord26 opened the pull request:
Offending melee weapons now deal normal damage, which will be clarified in a subsequent pull. Explosive damage is now Explosive damage and not also Concussive damage, which is dead.
#491 
EDIT: now also involves #577 via sub-pull.

Turtlelord26 committed:
Melee weapons no longer have explosive damage

Sledgehammer and Power Fist now deal regular ballistic damage ... or would if any melee weapon specified its damage type.

Scope crept on instances of skill being used in damage calculations of thrown weapons. They are gone now, per #508

Turtlelord26 committed:
Clarify Explosive damage

Is now not also concussive damage. 'Concussive' as a damage is now effectively deleted from Compound X. Individual effects that want to do unusual things re armor but are not explosive must be given prose rule exception text in their notes sections.

bleehu commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -169,7 +166,7 @@ THROWN WEAPONS
 == Gurber Throwing Knife $80 ==
 
-Damage: 15 + STR + DEX + [Thrown Weapon Skill]/2 + 1xBleed
+Damage: 15 + STR + DEX + 1xBleed
```
This constitutes a nasty damage debuff. Can we up the base damage some?

bleehu commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -185,7 +182,7 @@ AP Level: 2
 == EAL Tactical Throwing Axe $160 ==
 
-Damage: 25 + 3*STR + DEX + [Thrown Weapon Skill]/2 + 1xBleed
+Damage: 25 + 3*STR + DEX + 1xBleed
```
Same here; this was also written before we limited skills to a 6 point/level cap. We need to amp the damage on each of these by a bit.

bleehu reviewed (changes requested):
Simply removing the skill damage bonus from thrown weapons without putting any damage back is pulling them too far below the power curve. Simple fix: also bump their damage by around 15-20 points depending on level of weapon.

Turtlelord26 commented:
Well, I've been meaning to go over all of the melee weapon damages. Will make a PR to this PR.

rosebrallier commented on 01_04_Damage_Types_and_Armor_Types.txt:
```diff
@@ -22,10 +22,10 @@ ballistic armor is hit with a laser, they receive damage as if they weren't
-Damage ignores armor and shields (explosive damage is essentially, a crushing 
-force). Fragmentation grenades are considered ballistic weapons, as their 
-primary damage delivery method is shrapnel. 
+    Certain grenades and devices have Explosive damage, which ignores armor and 
```
Scope creep: Do we want to standardize to weapons "having" or "dealing" damage types? I personally prefer "dealing".

Turtlelord26 replied:
Agree

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -117,11 +117,9 @@ Range: 2m
-before physical damage if the target's armor has defense against explosive (i.e. 
-concussive) damage. If armor roll is greater than 50, target rolls a DC 50 
+	If armor roll is greater than 50, target rolls a DC 50 
 fort save for a broken limb of your choice. -20 stealth rolls for being quiet 
```
Scope creep: needs more clarity re results of save on success vs failure
May want to specify that the "you" here is the wielder.
What even are our rules for broken limbs?

Turtlelord26 replied:
We don't have them. This weapon is ancient.

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -117,11 +117,9 @@ Range: 2m
+	If armor roll is greater than 50, target rolls a DC 50 
 fort save for a broken limb of your choice. -20 stealth rolls for being quiet 
-on stealth attacks. Reduces door-breaking DC by 30 per hit.
+on stealth attacks. Reduces break DCs of doors by 30 per hit.
```
"break DCs" is super awk wording. Re my conventions this should become "the DC to break down a target door"

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -131,11 +129,10 @@ Range: Adjacent (1m)
-for 1 turn. -30% stealth rolls for being quiet on stealth attacks. Reduces 
-door-breaking DC by 45% per hit. This weapon has a Guard modifier of 0.
+	If armor roll is less than 50, target must pass a DC 50 fort save 
+or be stunned for 1 turn. -30 stealth rolls for being quiet on stealth 
```
Note to self: "-30 to Stealth checks"? Is Stealth a skill that I've specified? B/c if not I should do that. Also occurs above.

Turtlelord26 replied:
https://github.com/bleehu/Compound_X/blob/dev/CharacterCreation/06_Skills.txt#L75

rosebrallier replied:
Oh good! I... maybe need to use the sleeps.

rosebrallier reviewed (approved):
One sentence structure note, plus comments I'll happily make into new issues. Later.
Shipit pending blee's balance comments.

Turtlelord26 committed:
One sentence structure note

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -131,11 +129,10 @@ Range: Adjacent (1m)
-door-breaking DC by 45% per hit. This weapon has a Guard modifier of 0.
+	If armor roll is less than 50, target must pass a DC 50 fort save 
+or be stunned for 1 turn. -30 stealth rolls for being quiet on stealth 
+attacks. Reduces door-breaking DC by 45 per hit. This weapon has a 
```
Nother door reference. Also Stealth.

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -117,11 +117,9 @@ Range: 2m
-fort save for a broken limb of your choice. -20 stealth rolls for being quiet 
-on stealth attacks. Reduces door-breaking DC by 30 per hit.
+	If armor roll is greater than 50, target rolls a DC 50 Fort save 
+for a broken limb of your choice. -20 to stealth rolls for being quiet 
```
Is the Fort save to avoid the borked limb, keep composure at the borken ouch, smth else? We should be explicit abt the results of successful and failed saves.

rosebrallier reviewed (commented):
Saw more thing.

Turtlelord26 committed:
Typography

oops.

Turtlelord26 committed:
Added Welding Torch, added #s to descriptions

Added Eliazar Welder, identical to the Plasma Torch but with Fire damage type, reduced cost, and increased working time. Bumped Plasma Torch to lv 5 for more consistency in when players have access to plasma damage.

Turtlelord26 committed:
Update Melee Damage Calculations, add Damage Types

Design consistencies.

Daggers should do about 50 damage for someone with 7 Dex & Str (mods of 8), Swords 80.
Nonexotic Light weapons have a maximum total attribute modifier of 2, One-handed 3. Two-handed can go higher with Str mutipliers.
* Tools like torches get no attribute bonuses.
* Daggers receive 2x Dex but no Str, 
* Swords and Axes get 2x Str and 1x Dex
* Spears and hammers, for example, I would rate at 3x Str, no Dex

Throwing weapons set approx equivalent to their melee counterparts, dropping a level of STR mod if applicable. I gave the Axes +10 damage compared to to the knives.

Specific Notes:
The grabber knife lost 10 damage from the new baseline for its ability to start free grapples. I reworked the rest to be more streamlines.
The whip's target damage is 66.
The Fist and sledge hammer kept their obnoxiously high Str multipliers, but had base damage reduced to keep the ceiling reasonable.

Turtlelord26 committed:
Cut Sword ranges from 2 to 1m

Melee weapons, barring exotic outliers and specific reach weapons, should all have range 1.

Turtlelord26 committed:
Review notes: Put APL back in Thrown

+ misc and word choice.

Turtlelord26 committed:
Moved Damage Type after attribute mods

Also gave the Power Fist and Sledgehammer minimum damage.

Turtlelord26 committed:
Review Typography

Turtlelord26 committed:
Merge pull request #630 from bleehu/Turtlelord26-Melee_Weapon_Damage_Patch

Turtlelord26 melee weapon damage patch

Turtlelord26 committed:
Word Choice and Typography

bleehu reviewed (approved):
shipit!

Turtlelord26 committed:
Merge branch 'dev' into Turtlelord26-#491-Explosive_Damage_Consistency
